bitmark-references/bitmark-node-software/bitmark-blockain.md

```diff
@@ -0,0 +1,166 @@
+# Basic Bitmark Blockchain Overview
+
+This briefly describe the structure of the block chain. It builds up
+from the transactions through to the final structure of the chain
+itself.
+
+## Definitions
+
+The following items are used in the descriptions below.
+
+Byte
+: A single binary byte.
+
+Unn
+: An unsigned integer of "nn" bits length.
+
+VarInt
```

Comment:
Do we need to give some examples?
Example: https://developers.google.com/protocol-buffers/docs/encoding